Game/ThingFactory.pde

```diff
@@ -0,0 +1,6 @@
+class ThingFactory {
+
+  Thing createWall(int p, int r) {
+    return new Thing(createShape(BOX, Room.TILE, 800, Room.TILE), p, r);
```

Comment:
Use Room.ROOM_HEIGHT. I just made it static and pushed the change to master.

Reply:
ah, very nice :)